### ISSUE TITLE
Multi-platform runtime loader does not work for relative imports

### ISSUE THREAD

**Issue by joonis** [dashingsoft/pyarmor#1238]
For relative imports the import level must be set to `1`, but it is set to `0`:

```
for suffix in '', '_a1', '_a2', '_a3':
    try:
        __pyarmor__ = __import__('pyarmor_runtime' + suffix,
                                 globals(), locals(),
                                 ('__pyarmor__',),
                                 0).__pyarmor__
        break
    except ModuleNotFoundError:
        pass
else:
    raise ModuleNotFoundError('no pyarmor_runtime extension found')
```

Furthermore this import strategy is really implicit. Finally you don't know what binary is for which platform. Why don't you use the same folder structure as used in Pyarmor 7?

**Comment by jondy** [comment]
My original idea is by this way it need not check the different machine, but I think you're right, it's not a good idea.

I'll refine it with different folder for each platform.

**Comment by jondy** [comment]
Fixed in 8.2.1
